apps/web/lib/app-providers-app-dir.tsx

```diff
@@ -287,7 +287,7 @@ const AppProviders = (props: PageWrapperProps) => {
     RemainingProviders
   );
 
-  if (isBookingPage) {
+  if (props.isBookingPage || isBookingPage) {
```

Comment:
correct me if I'm wrong. Do we still need it like this? or

```suggestion
  if (props.isBookingPage) {
```

like this?

Reply:
let's continue using `useIsBookingPage()` hook for now! (until we migrate all routes, which need `isBookingPage` flag to be true, to `(booking-page-wrapper)` route group)